tests/test_retrievers_langchain_pathway.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
```

Comment:
Is it passable to separate this test into 2 scripts? Based on the CI logical, when the files under `comps/vectorstores/langchain/pathway` changed, the test `test_vectorstores_langchain_pathway.sh` will be triggered. But now there is only `test_retrievers_langchain_pathway.sh`.   
You can also copy this test scripts into `test_vectorstores_langchain_pathway.sh`.

Reply:
I created a new test for it. It's just a trimmed version of this retriever test without the `retriever` microservice part.

Reply:
In addition, I iterated over your other comments, thanks. 
If this is ok now, I can also resolve this.